gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopOutputStreamTest.java

```diff
@@ -278,6 +283,89 @@ public void hsync_statistics() throws IOException {
     assertThat(fout.getIOStatistics().counters().get(INVOCATION_HFLUSH.getSymbol())).isEqualTo(1);
   }
 
+  @Test
+  public void hsync_increment_timeStatistics() throws IOException {
+    Path objectPath = new Path(ghfs.getUri().resolve("/dir/object2.txt"));
+    FileSystem.Statistics statistics = new FileSystem.Statistics(ghfs.getScheme());
+    GoogleHadoopOutputStream fout =
```

Comment:
consider reducing code duplication in this file

Reply:
Created a single function to test all 3 statistics